ethcore/src/spec/spec.rs

```diff
@@ -163,13 +250,14 @@ impl Clone for Spec {
 			seal_rlp: self.seal_rlp.clone(),
 			hardcoded_sync: self.hardcoded_sync.clone(),
 			constructors: self.constructors.clone(),
-			state_root_memo: RwLock::new(*self.state_root_memo.read()),
+			state_root_memo: self.state_root_memo,
 			genesis_state: self.genesis_state.clone(),
 		}
 	}
 }
 
 /// Part of `Spec`. Describes the hardcoded synchronization parameters.
+#[derive(Debug, Clone)]
```

Comment:
`Clone` replaces `impl Clone` bolierplate.
`Debug` is has replaced `to_json` which was used to print this structure